### PR TITLE
Implement better ActiveSupport notification span logging.

### PR DESCRIPTION
**Pull request by skirushkin**
- Correct JSON for `context` field (ActiveSupport payload)
- `message` defines by ActiveSupport event name
- Removed unnecessary JCW::Tracing with subscribers.
- Do not produce ActiveSupport::Notifications::Event.new(...) on each event (less memory)
- Correct filter for big objects in `*.action_controller` events.
- Correct README.md (with notification about memory leak)
- Bump nokogiri
- Version bump

### CODE REVIEW

**Review by nulldef — APPROVED** [review]

File: lib/jcw/subscriber.rb [hunk 1]
```diff
@@ -1,26 +1,29 @@
...
-        {
-          name: event.name,
-          time: event.time,
-          payload: event.payload.to_s,
-          transaction_id: event.transaction_id,
-        }.as_json
+      if data.is_a?(Hash)
+        # we should only mutate the copy of the data
+        data = data.dup
+        IGNORED_DATA_TYPES.each { |key| data.delete(key) if data.key?(key) } # cleanup data
```
Comment (tycooon) [inline lib/jcw/subscriber.rb:25]: Там получается в массиве ключи, а не типы лежат? 🤔

Comment (skirushkin) [reply lib/jcw/subscriber.rb:25]: Ага, можно переименовать как-нибудь поумнее 🤔

Comment (skirushkin) [reply lib/jcw/subscriber.rb]: `IGNORED_DATA_KEYS` вроде бы идеально, не?

Comment (tycooon) [reply lib/jcw/subscriber.rb]: 👍

**Pull request merged**